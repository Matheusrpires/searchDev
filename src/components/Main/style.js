import styled from "styled-components";

export const Wrapper  = styled.main`
  background-color: #78be20;
  color: red;
`;

