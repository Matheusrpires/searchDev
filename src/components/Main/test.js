import React from "react";
import { render, screen } from "@testing-library/react";
import 'jest-styled-components'
import Main from ".";

describe("<Main />", () => {
  it("should render the heading", () => {
    const { container } = render(<Main />);

    expect(
      screen.getByRole("heading", { name: /react avanÃ§ado/i })
    ).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });

  it("should render background red", () => {
    const { container } = render(<Main />);

    expect(container.firstChild).toHaveStyle({ color: "red" });
  });
});
