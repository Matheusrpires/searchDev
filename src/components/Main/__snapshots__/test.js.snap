// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render the heading 1`] = `
.c0 {
  background-color: #78be20;
  color: red;
}

<main
  class="c0"
>
  <h1>
    React Avan√ßado
  </h1>
</main>
`;
